anomdetect/src/test/java/com/expedia/adaptivealerting/anomdetect/forecast/point/PewmaPointForecasterTest.java

```diff
@@ -66,70 +60,57 @@ public void pewmaCloseToEwmaWithZeroBeta() throws IOException {
         val pewmaParams = new PewmaParams()
                 .setAlpha(DEFAULT_ALPHA)
                 .setBeta(beta)
-                .setWeakSigmas(WEAK_SIGMAS)
-                .setStrongSigmas(STRONG_SIGMAS)
                 .setInitMeanEstimate(observed0);
-        val pewmaDetector = new PewmaDetector();
-        pewmaDetector.init(detectorUuid, pewmaParams, AnomalyType.TWO_TAILED);
+        val pewmaPointForecaster = new PewmaPointForecaster(pewmaParams.toPointForecasterParams());
 
         val ewmaParams = new EwmaParams()
                 .setAlpha(DEFAULT_ALPHA)
-                .setWeakSigmas(WEAK_SIGMAS)
-                .setStrongSigmas(STRONG_SIGMAS)
                 .setInitMeanEstimate(observed0);
-        val ewmaDetector = new EwmaDetector();
-        ewmaDetector.init(UUID.randomUUID(), ewmaParams, AnomalyType.TWO_TAILED);
+        val ewmaPointForecaster = new EwmaPointForecaster(ewmaParams.toPointForecasterParams());
 
         int rowCount = 1;
         while (testRows.hasNext()) {
             val observed = Float.parseFloat(testRows.next()[0]);
-
-            val ewmaStdDev = sqrt(ewmaDetector.getVariance());
+//            val ewmaStdDev = sqrt(ewmaDetector.getVariance());
```

Comment:
Remove or reinstate comments in this method

Reply:
Done